### PR TITLE
added telemetry_infrastructure_logs_bucket variable

### PR DESCRIPTION
**Pull request by kostz**
https://github.bus.zalan.do/logging/logging-platform/issues/545

### CODE REVIEW

**Comment by kostz** [comment]
👍

**Comment by aermakov-zalando** [comment]
Normally we use `logging_` for the config items manage by your team, which would then allow you to change them in Cluster Registry. Do you want to use a different prefix from now on?

**Comment by kostz** [comment]
> Normally we use `logging_` for the config items manage by your team, which would then allow you to change them in Cluster Registry. Do you want to use a different prefix from now on?

that's a tricky thing since from one hand `logging` team was recently renamed to `telemetry-infrastructure`, but from other hand I agree that in this case maybe it's better to support legacy conventions with `logging_` prefix, we'll discuss this internally and come back soon

anyway am I correct that it's only naming and nothing really relies on that names, like write policy, any filters, etc, or there's something that I'm not aware of?

**Comment by aermakov-zalando** [comment]
As I've said in the comment, `logging_*` keys are available for your team to modify in Cluster Registry (along with `zmon_`, `tracing_` and `scalyr_`). If you want a new prefix, we will need to add it to the list as well.

**Comment by mikkeloscar** [comment]
This needs to be rebased on `dev`

File: cluster/cluster.yaml [hunk 1]
```diff
@@ -1569,6 +1569,24 @@ Resources:
...
+            - Action: ["s3:ListBucket"]
+              Effect: "Allow"
+              Resource: 'arn:aws:s3:::{{.Cluster.ConfigItems.logging_infrastructure_s3_bucket}}'
+            - Action: ["s3:GetObject"]
+              Effect: "Allow"
+              Resource: 'arn:aws:s3:::{{.Cluster.ConfigItems.logging_infrastructure_s3_bucket}}/*'
+            - Action: ["s3:PutObject"]
+              Effect: "Allow"
+              Resource: 'arn:aws:s3:::{{.Cluster.ConfigItems.logging_infrastructure_s3_bucket}}/*'
+            - Action: [ "s3:PutObjectAcl" ]
```
Comment (aermakov-zalando) [inline cluster/cluster.yaml]: Can we merge these into the same statement?

File: cluster/cluster.yaml [hunk 2]
```diff
@@ -1569,6 +1569,24 @@ Resources:
             - Action: ["s3:PutObject"]
               Effect: "Allow"
               Resource: 'arn:aws:s3:::{{.Cluster.ConfigItems.logging_s3_bucket}}/*'
+{{- if ne .Cluster.ConfigItems.telemetry_infrastructure_logs_bucket "" }}
```
Comment (aermakov-zalando) [inline cluster/cluster.yaml]: Should be renamed here as well.

Comment (kostz) [reply cluster/cluster.yaml]: yes, that was corrected

File: cluster/cluster.yaml (hunk 1 above)
Comment (kostz) [reply cluster/cluster.yaml]: merged

**Comment by kostz** [comment]
👍

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**